src/app/views/provisioning/order.controller.coffee

```diff
@@ -84,6 +84,7 @@
         )
       .finally(() -> vm.isLoading = false)
   else
+    vm.isLoading = true
```

Comment:
I think you can extract this out of the conditional branches to avoid duplication